ideascube/forms.py

```diff
@@ -52,3 +60,37 @@ def save(self, *args, **kwargs):
         user.is_staff = True
         user.save()
         return user
+
+
+class UserImportForm(forms.Form):
+    source = forms.FileField(required=True)
+
+    def save(self):
+        source = self.cleaned_data['source']
+        users = []
+        errors = []
+        rows = csv.DictReader(source.read().decode().splitlines())
```

Comment:
I do like this approach, but we'll have the issue to convert to unicode. We may create a generic generator function that does that (calling TextIOWrapper, itering over it, and calling to_unicode for each row). What do you think?